lexicons/com/atproto/temp/fetchLabels.json

```diff
@@ -0,0 +1,35 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.temp.fetchLabels",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Fetch all labels from a labeler created after a certain date.",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "since": { "type": "integer" },
```

Comment:
The response labels will have `cts` as a datetime string, so caller will need to parse those and turn them in to UNIX timestamp for next request.

I know @whyrusleeping prefers UNIX timestamps, but wanted to double-check that whether is actually more ergonomic in this case.

I did a quick grep and didn't see any use of a UNIX timestamps in the current lexicons. On the other hand, don't want to hold things up: if why is :+1: on it this way than no problem from me on a temp endpoint.

Reply:
Checked in with why & he said UNIX timestamps are better for him.

I'll just stick with that but we can reconsider when it comes around to the official api